src/containers/pages/InterventionPlan/IRS/plan/index.tsx

```diff
@@ -110,7 +112,7 @@ export interface IrsPlanProps {
   isDraftPlan?: boolean;
   isFinalizedPlan?: boolean;
   isNewPlan?: boolean;
-  jurisdictionsArray: Jurisdiction[];
+  jurisdictionsById: { [key: string]: Jurisdiction };
```

Comment:
Do we want to necessarily replace `jurisdictionsArray `like this, or just add `jurisdictionsById` alongside it?

Reply:
Yep want to completely remove it, there should be no need tor it at all at this point!